docs/custom_quantum_functions.md

```diff
@@ -135,6 +151,29 @@ void led_set_user(uint8_t usb_led) {
 * Keyboard/Revision: `void led_set_kb(uint8_t usb_led)`
 * Keymap: `void led_set_user(uint8_t usb_led)`
 
+## `host_keyboard_leds()`
+
+Call this function to get the last received LED state.
+This is useful for reading the LED state outside `led_set_*`, e.g. in [`matrix_scan_user()`](#matrix-scanning-code).
+
+For convenience, you can use the `IS_HOST_LED_ON(LED_NAME)` and `IS_HOST_LED_OFF(LED_NAME)` macros instead of calling directly `host_keyboard_leds()`.
+
+## Setting physical LED state
+
+Some keyboard implementations provide convenience methods for setting the state of the physical LED's.
```

Comment:
```suggestion
Some keyboard implementations provide convenience methods for setting the state of the physical LEDs.
```

Reply:
I have replaced “_LED's_” with “LEDs” everywhere in the file.

Note that there are other documentation files as well as some code files (mainly in keymaps/keyboards) where “LED's” is also used.